frontend/apps/reader/modules/readerview.lua

```diff
@@ -211,6 +211,9 @@ function ReaderView:paintTo(bb, x, y)
             local center_offset = bit.rshift(self.arrow.height, 1)
             -- Paint at the proper y origin depending on wheter we paged forward (dim_area.y == 0) or backward
             self.arrow:paintTo(bb, 0, self.dim_area.y == 0 and self.dim_area.h - center_offset or self.dim_area.y - center_offset)
+        elseif self.page_overlap_style == "line" then
+            bb:paintRect(0, self.dim_area.y == 0 and self.dim_area.h or self.dim_area.y,
+                self.dim_area.w, 2, Blitbuffer.COLOR_BLACK)
```

Comment:
Or instead of 2, one of (or a new among them) the `Size.line.medium/thick` ?

Reply:
We have this "2" in highlighting drawer and marker too:
https://github.com/koreader/koreader/blob/2ba694d394b0b2ea490d83f6370f42ff257c584b/frontend/apps/reader/modules/readerview.lua#L614-L632

Reply:
That probably just means we overlooked it at the time. This does look like a Size.line.medium situation. :-)